https://bugs.gentoo.org/453778

we've had a stable util-linux that supports -s for a long time (and the ebuild
forces a recent version), so disable this test which implicitly tries to modify
files like /etc/mtab and piss off the sandbox.

--- a/aclocal.m4
+++ b/aclocal.m4
@@ -65,7 +65,7 @@
 AC_DEFUN(AF_SLOPPY_MOUNT,
 [if test -n "$MOUNT" ; then
   AC_MSG_CHECKING([if mount accepts the -s option])
-  if "$MOUNT" -s > /dev/null 2>&1 ; then
+  if true > /dev/null 2>&1 ; then
     AC_DEFINE(HAVE_SLOPPY_MOUNT, 1, [define if the mount command supports the -s option])
     AC_MSG_RESULT(yes)
   else
